build/check_build.py

```diff
@@ -1,12 +1,15 @@
 import sys
 import os
 import xml.etree.ElementTree
+import glob
+import fnmatch
 
 SUCCESS = 0
 INCORRECT_ARGS = 1
 NO_BUILD_PROPERTIES = 2
 ICONS_NOT_INCLUDED = 3
 ICON_LOCATION_POINTER_WRONG = 4
+TEST_CLASSES_NOT_RUNNING = 5
```

Comment:
This seems to be unused. Was it supposed to be a failure code?

Reply:
Yes, it was supposed to check if there were any tests that fit the naming conventions, but didn't have any @Test decorator in them, but actually it's outside of scope for the ticket I think, so not necessary.

Reply:
If it is not necessary it should be removed.

Reply:
Okay, I've actually made a change so it is used, and then build.bat will stop if sent an exit code other than 0; I think this is closer to what was originally intended by the ticket, rather than only doing a print().